### PR TITLE
Indentation

### PR DESCRIPTION
**Pull request by tfettig22**
What does this PR do?
* Adjust indentation to be a consistent 2-space indent across all files
* Add semi colons throughout

### CODE REVIEW

**Review by aspitz1 — APPROVED** [review]
Thank you! Looks good.

File: cypress/e2e/error-handling-spec.cy.js [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
+    cy.get('.message')
+      .contains('Oops... the requested prescription is not on file. Check the URL or contact your taste physician.');
+  });
 
-    it('Should have 404 page for "garbage" URL', () => {
-        cy.visit('http://localhost:3000/gramcracker')
-            .get('.not-found-message')
-            .contains('404: Not found');
-    });
+  it('Should have 404 page for "garbage" URL', () => {
```
Comment (edjrodriguez) [inline cypress/e2e/error-handling-spec.cy.js:45]: Should we add a test here that looks for the "Back" button and clicks on it to go back to main?  That is technically part of the user flow

**Review by edjrodriguez — COMMENTED** [review]
Such attention to detail.  Nice and tidy!

**Review by dstavis — APPROVED** [review]
Love it

File: cypress/e2e/error-handling-spec.cy.js (hunk 1 above)
Comment (dstavis) [reply cypress/e2e/error-handling-spec.cy.js:45]: @aspitz1 What do you think- is it too late, or worth doing?

**Pull request merged**